app/checkstyle.gradle

```diff
@@ -41,10 +41,14 @@ task checkstyle(type: Checkstyle) {
     configFile file('../config/checkstyle/checkstyle.xml')
     source 'src'
     include '**/*.java'
-    exclude '**/gen/**'
+    //exclude '**/gen/**'
```

Comment:
This should be deleted if it isn't needed.

Reply:
I deleted it